py/torch_tensorrt/_Input.py

```diff
@@ -352,9 +353,20 @@ def example_tensor(
                 )
             else:
                 if isinstance(self.shape, tuple):
-                    return torch.rand(self.shape).to(
-                        dtype=self.dtype.to(torch.dtype, use_default=True)
-                    )
+                    if self.dtype in [dtype.u8, dtype.i8, dtype.i32, dtype.i64]:
```

Comment:
It appears that the changes ensure values are within the range of -128 to 127 for `dtype.i8`, `dtype.i32`, and `dtype.i64`, and within the range of 0 to 127 for `dtype.u8`. 

Previously, it seems there was an issue where casting random values between 0 and 1 to integers resulted in most values being 0.

This fix seems necessary and addresses the problem effectively.

Reply:
Yes, but It caused problem in model test as random int value was used in fake tensor as index.
Added option to initialize it with zero.